engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -168,8 +172,11 @@ class WorkflowManagerActor(config: Config)
 
   when (Running) {
     case Event(SubmitWorkflow(source), _) =>
-      val updatedData = submitWorkflow(source, replyTo = Option(sender), id = None)
-      stay() using updatedData
+      submitWorkflow(source, replyTo = Option(sender), id = None) onSuccess {
```

Comment:
`onSuccess` is going to be deprecated in the next Scala release, use `map` instead